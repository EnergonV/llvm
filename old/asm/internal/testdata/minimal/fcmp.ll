; ModuleID = 'fcmp.c'
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

; Function Attrs: nounwind uwtable
define i32 @main() #0 {
  %a = alloca float, align 4
  %b = alloca float, align 4
  %1 = load float, float* %a, align 4
  %2 = load float, float* %b, align 4
  %3 = fcmp olt float %1, %2
  %4 = zext i1 %3 to i32
  ret i32 0
}

attributes #0 = { nounwind uwtable "disable-tail-calls"="false" "less-precise-fpmad"="false" "no-frame-pointer-elim"="true" "no-frame-pointer-elim-non-leaf" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+fxsr,+mmx,+sse,+sse2" "unsafe-fp-math"="false" "use-soft-float"="false" }

!llvm.ident = !{!0}

!0 = !{!"clang version 3.8.0 (tags/RELEASE_380/final)"}
