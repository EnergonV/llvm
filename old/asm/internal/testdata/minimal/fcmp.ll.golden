define i32 @main() {
; <label>:0
	%a = alloca float
	%b = alloca float
	%1 = load float, float* %a
	%2 = load float, float* %b
	%3 = fcmp olt float %1, %2
	%4 = zext i1 %3 to i32
	ret i32 0
}
